base_layer/core/src/base_node/states/block_sync.rs

```diff
@@ -78,6 +81,65 @@ impl Default for BlockSyncConfig {
     }
 }
 
+#[derive(Clone, Copy, Debug)]
+pub enum BlockSyncStrategy {
+    ReverseViaBestChainMetadata(BestChainMetadataBlockSyncInfo),
```

Comment:
I do not believe this is an accurate description of the current sync strategy, it only performs forward syncing. A better name would be `ForwardViaBestChainMetadata` or `ForwardWithChainSplitDetection` or `ForwardWithChainSplitDetectionViaBestChainMetadata`

Reply:
Updated to `ViaBestChainMetadata`